src/select.test.visuals.ts

```diff
@@ -47,33 +47,67 @@ for (const story of stories) {
           );
         });
 
-        test('open=${true}', async ({ page }, test) => {
-          await page.goto(`?id=${story.id}&globals=theme:${theme}`);
-
-          await page
-            .locator('glide-core-select')
-            .evaluate<void, Select>((element) => {
-              element.open = true;
-            });
-
-          await page
-            .locator('glide-core-option glide-core-menu')
-            .evaluate<void, Menu>((element) => {
-              element.open = true;
-            });
-
-          await expect(page).toHaveScreenshot(
-            `${test.titlePath.join('.')}.png`,
-          );
+        test.describe('multiple=${false}', () => {
```

Comment:
Wrapped a few existing tests in `test.describe('multiple=${false}'` and  `test.describe('multiple=${true}'`.